frontend/src/assets/css/variables.scss

```
@@ -1,7 +1,11 @@
 :root {
+  --regular: 400;
+  --medium: 500;
+  --semibold: 600;
   --white: #ffffff;
   --shadow: 0 2px 4px #0000001a;
   --black: #000000;
+  --transparent: #00000000;
```

let's remove variables for transparent and font-weights
```suggestion
  --white: #ffffff;
  --shadow: 0 2px 4px #0000001a;
  --black: #000000;
```

---

done